blog/2023-09-08-newsletter.md

```diff
@@ -0,0 +1,185 @@
+
+[**Conda Build 3.26.0**](https://conda.org/blog/2023-07-28-july-2023-releases#changes-in-conda-237023712372): Logs extra-meta data for package verification, adds pip to env-doc make command, and fixes various bugs related to handling unknown binaries and Git cloning with submodules.
+
+[**constructor 3.4.5**](https://github.com/conda/constructor/releases/tag/3.4.5): Constructor 3.4.5 enhances support for .yaml and .txt files in the environment_file key while ensuring accurate conda metadata in installer creation from existing environments on Windows.
+
+[**conda-libmamba-solver 23.7.0**](https://github.com/conda/conda-libmamba-solver/releases/tag/23.7.0): Libmamba Solver 23.7.0 introduces local channel integrations in conda-build, fixes for namespaced settings, CLI argument handling, authentication in package downloads, and improved handling of channels in conda-lock, among other enhancements and bug fixes.
+
+[**conda-lock 2.1.2**](https://github.com/conda/conda-lock/releases/tag/v2.1.2): conda-lock 2.1.2 resolves Pydantic v2 warnings, implements auth stripping for private PyPi packages, and enhances compatibility with Mamba>=1.4.6. Additionally, it includes infrastructure improvements, dependency updates to pydantic 2.0, and the removal of deprecated pkg_resources, among other changes.
+
+[**conda-package-streaming 0.9.0**](https://github.com/conda/conda-package-streaming/releases/tag/v0.9.0)
+
+[**conda-package-handling 2.2.0**](https://github.com/conda/conda-package-handling/releases/tag/2.2.0)
+
+[**Pixi**](https://github.com/prefix-dev/pixi/releases/tag/v0.2.0): Pixi 0.2.0 introduces a new "pixi search" command for package searching and target-specific tasks. It also includes several bug fixes, installation improvements, and enhancements like a reduction in binary size, updated banner, and contribution guidelines.
+
+---
+
+## [Job still Open at sfbi!](https://www.sfbi.fr/emplois/offre/202308020700-cdd-ingenieur-e-en-developpement-logiciel)
+
+**Job Role**: Software Development Engineer
+
+**Contract Type**: Fixed-Term (12 months)
+
+**Location**: Versailles, France
+
+**Starting Date**: October 1, 2023
+
+**Education**: Bac+5 / Master's Degree
+
+Join in revolutionizing genomics research! They're looking for a Software Development Engineer to enhance our flagship tool, REPET. 
+
+**Your mission**: improve portability and scalability. If you have skills in Python, conda, Snakemake, and more, apply by September 20, 2023. [**Learn more**](https://www.sfbi.fr/emplois/offre/202308020700-cdd-ingenieur-e-en-developpement-logiciel).
+
+---
+
+<Image img={yt} alt="Image of the talk on Youtube"/>
+
+## [Writing Plugin Friendly Python Applications: by Travis](https://youtu.be/d40tBcqopAI?feature=shared)
```

Comment:
I think this should read "...by conda maintainer Travis Hathaway".

First, in the conda community, there is actually a pretty famous Travis already (Travis Oliphant). He was pivotal in the creation of the conda tool, so I don't want readers to be confused.

Second, adding that information there gives more context to the reader who probably has no idea who I am.

Reply:
Okay!
Let me know if "Writing Plugin Friendly Python Applications: by Travis ( A Conda Maintainer)", works for you!

Reply:
"Writing Plugin Friendly Python Applications: by Travis Hathaway ( A Conda Maintainer)"

You need to use full names.

Reply:
Done!